### PR TITLE
Updated Survey Form UI

### PR DESCRIPTION
**Pull request by sshsrn**
Signed-off-by: SRIHARI S <sshsrn@gmail.com>

### CODE REVIEW

**Comment by SSHSRN** [comment]
I have updated the UI of the survey form.

Before:
image

After:
image

**Comment by SSHSRN** [comment]
@praveenscience Kindly check!

File: surveyform.html [hunk 1]
```diff
@@ -29,20 +29,40 @@
...
     </style>
   </head>
   <body>
     <h1 class="heading"><em>WELCOME TO THE SHORT SURVEY</em></h1>
     <form>
       <div>
-        <h2 id="blue">About yourself</h2>
-        <label for="uname">Enter your name:</label>
-        <input type="text" name="uname" id="uname" placeholder="Enter your name" />
+        <center><h2 id="blue">About yourself</h2></center>
```
Comment (praveenscience) [inline surveyform.html:59]: This tag is deprecated. Can we remove this and use CSS?

File: surveyform.html [hunk 2]
```diff
@@ -63,21 +84,21 @@ <h2 id="blue">About yourself</h2>
...
       <input type="radio" name="rate" value="4" />
       <input type="radio" name="rate" value="5" /> Excellent
       <fieldset>
-        <legend>Any comments</legend>
-        <textarea name="comments" id="comm" cols="60" rows="10"></textarea>
+        <legend>Any other comments</legend>
+        <textarea name="comments" id="comm" cols="60" rows="10" style="resize: none; overflow: scroll"></textarea>
       </fieldset>
-      <input type="submit" value="Submit" />
+      <center><input class="submitBtn" type="submit" value="Submit" /></center>
```
Comment (praveenscience) [inline surveyform.html:101]: This tag is deprecated. Can we remove this and use CSS?

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by praveenscience** [comment]
Any updates? It's the last few hours of Hacktoberfest? 😁

**Pull request closed without merging**